course/frontend/has-symbol/test/hassymbolSpecs.js

```diff
@@ -0,0 +1,37 @@
+describe('has symbol test', function () {
```

Comment:
hey! please rename hassymbolSpect.js to  estandar name `hassimbol**.spec.**js`

Reply:
Hi, thanks for the advice !!! but this is a little description, not the name itself.